Switch actor list sides

ProfessorFess opened the pull request:
Lets "q" switch the actor list to the left and "Q" to switch it back to the right

ProfessorFess committed:
Allow "£" to open help menu

ProfessorFess committed:
Merge pull request #1 from ProfessorFess/ProfessorFess-patch-1

Allow "£" to open help menu

ProfessorFess committed:
Add switch actor list side functions

Adds functions to have "h" move the actor list to the left and "H" move the list back to the right

14to9 commented on js/app.js:
```diff
@@ -608,6 +616,25 @@ $(function(){
+    switchActorListRight: function() {
+      document.getElementById("actorapp").style.cssFloat = "right";
+      document.getElementById("fixed-panel").style.cssFloat = "left";
+      document.getElementById("fixed-panel").style.left = "0px";
```
Let's extract a variable for the fixed-panel div?

14to9 commented on js/app.js:
```diff
@@ -608,6 +616,25 @@ $(function(){
+      document.getElementById("actorapp").style.cssFloat = "left";
+      document.getElementById("fixed-panel").style.cssFloat = "right";
+      document.getElementById("fixed-panel").style.left = "287px";
+    },
```
Let's extract a function to DRY (don't repeat yourself) up these two behaviors?

14to9 commented on js/app.js:
```diff
@@ -608,6 +616,25 @@ $(function(){
+      actor.removeCondition(Condition.newCondition("happy"));
+     }
+     Actors.each(f);
+    },
```
happy behavior is undesired in this PR, please remove

14to9 commented on js/app.js:
```diff
@@ -320,6 +320,10 @@ $(function(){
+        case 113:  // 'q'
+          this.switchActorListLeft(); break;
+        case 81: // 'Q'
+          this.switchActorListRight(); break;
```
Let's bind this behavior to `\` and have it toggle on press, rather than bound to a pair of keys

14to9 reviewed (changes requested):
Some light refactorings please.

Also consider how to preserve layout selection beyond reload?

ProfessorFess committed:
Update

14to9 commented on js/app.js:
```diff
@@ -320,6 +323,8 @@ $(function(){
           this.removeLastConditionFromActor(Actors.selectedActor()); break;
         case 114:  // 'r'
           this.rotateConditions(Actors.selectedActor()); break;
+        case 92:  // '\'
```
Add this key to the help file text?

14to9 reviewed (approved):
add help text when you can?

ProfessorFess committed:
Add help text for switching layout

Adds a line of text onto the help menu popup for the switch actor list function
